vnext/template/cpp-app/src/App.cpp

```diff
@@ -53,7 +52,19 @@ void App::OnLaunched(activation::LaunchActivatedEventArgs const& e)
     super::OnLaunched(e);
 
     Frame rootFrame = Window::Current().Content().as<Frame>();
-    rootFrame.Navigate(xaml_typename<{{ namespaceCpp }}::MainPage>(), box_value(e.Arguments()));
+    rootFrame.Navigate(xaml_typename<MainPage>(), box_value(e.Arguments()));
+}
+
+/// <summary>
+/// Invoked when the application is activated by some means other than normal launching.
+/// </summary>
+void App::OnActivated(Activation::IActivatedEventArgs const &e) {
+  auto preActivationContent = Window::Current().Content();
```

Comment:
this would crash xaml islands apps since Window.Current is null
We may not have seen crashes on this since win32 apps don't get lifetime events but that will change with Reunion

Reply:
This is in the App code which I think is not applicable to the XAML islands. I think we can address this issue in future when we are getting closer to the Reunion.